### PR TITLE
[CPDLP-3082] Update swagger docs with participant profile endpoints

### PR DESCRIPTION
**Pull request by leandroalemao**
### Context

Ticket: [CPDLP-3082](https://dfedigital.atlassian.net/browse/CPDLP-3082)

After the endpoints have been added for we need to add the correct documentation for all endpoints for the participant endpoints.

### Changes proposed in this pull request

Update swagger docs with participant profile endpoints.

[CPDLP-3082]: https://dfedigital.atlassian.net/browse/CPDLP-3082?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: spec/swagger_helper.rb [hunk 1]
```diff
@@ -56,26 +58,18 @@
           Application: APPLICATION[version],
           ApplicationAcceptRequest: APPLICATION_ACCEPT_REQUEST,
           ApplicationChangeFundedPlaceRequest: APPLICATION_CHANGE_FUNDED_PLACE_REQUEST,
-        },
+          ApplicationsCsvResponse: APPLICATIONS_CSV_RESPONSE[version],
```
Comment (cwrw) [inline spec/swagger_helper.rb:61]: just a note that this has been done here https://github.com/DFE-Digital/npq-registration/pull/1400, could we make sure those align as that will probably be merged before?

Comment (leandroalemao) [reply spec/swagger_helper.rb:61]: Hi @cwrw Yeah I copied the same changes from there.. 👍

File: lib/rails_semantic_logger/extensions/action_dispatch/debug_exceptions.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# Log actual exceptions, not a string representation
```
Comment (cwrw) [inline lib/rails_semantic_logger/extensions/action_dispatch/debug_exceptions.rb:1]: do you mind keeping this out of this pr please, as it's in the other one?

Comment (leandroalemao) [reply lib/rails_semantic_logger/extensions/action_dispatch/debug_exceptions.rb]: Yeah sure tksss @cwrw 👍 😄

File: spec/swagger_schemas/models/participant.rb [hunk 3]
```diff
@@ -0,0 +1,513 @@
...
+                npq-early-years-leadership
+                npq-additional-support-offer
+                npq-early-headship-coaching-offer
+                npq-leading-primary-mathematics
+                npq-senco
+              ],
+            },
+          },
+          funded_places: {
+            description: "Information about the course(s) the participant is enroled in",
```
Comment (cwrw) [inline spec/swagger_schemas/models/participant.rb:63]: can we update this to funded places please?

**Review by cwrw — APPROVED** [review]

**Pull request merged**